asciidoc/sdpi-supplement-intro.adoc

```diff
@@ -20,7 +23,7 @@ These known limitations include:
 . *Requirements boxes* (e.g., "R1234") have been added especially in TF-2, with some also part of TF-1 and TF-3; this is an initial approach that *_will be significantly expanded in future versions of the supplement_*; an initial approach is provided in <<vol1_clause_sdpi_requirements_modeling_integration>>, with discussion related to how it will be expanded in future versions of the supplement;
 . *_<<term_safe_effective_secure>> Sections_* (see <<vol1_clause_appendix_a_ses_considerations_section_template>>) are included in the specification; however, their use and content will be significantly extended in SDPi 1.x and subsequent versions;
```

Comment:
Propose to make more future proof by changing "will be significantly extended in SDPi 1.x and subsequent versions" to "will be significantly extended in future versions"